alica_tests/src/test/etc/plans/behaviours/TestParameterPassingBehaviour.beh

```diff
@@ -10,5 +10,29 @@
   "relativeDirectory": "",
   "requiresParameters": true,
   "runtimeCondition": null,
-  "variables": []
+  "variables": [],
```

Comment:
Must manaully set relativeDirectory to `behaviours` at the moment or everything goes in the wrong place